scripts/random.coffee

```diff
@@ -18,18 +18,6 @@
 
 
 module.exports = (robot) ->
-  robot.respond /bash me/i, (msg) ->
```

Comment:
Just noticed. There's a difference between `robot.respond` and `robot.hear`. Respond requires the bot to be addressed before he writes back whereas hear acts on stuff that happens in the channel (for random jokes and stuff). Would it be possible to split these up again into two different tuples to act on?